src/FtlPlayer.ts

```diff
@@ -241,6 +241,11 @@ export class FtlPlayer {
                 this.janusPluginHandle?.createAnswer({
                     jsep: jsep,
                     media: { audioSend: false, videoSend: false, data: true },
+                    customizeSdp: function(jsep: JSEP) {
```

Comment:
Can we add a comment here explaining that this is a workaround for a Chromium bug with a link to the bug in question [here](https://bugs.chromium.org/p/webrtc/issues/detail?id=8133)?

Then I'd be happy to merge :)